### ISSUE TITLE
Move VEDA data types, utility functions to source code

### ISSUE THREAD

**Issue by hanbyul-here** [NASA-IMPACT/veda-ui#975]
## Context
This is a story that comes out from this epic ticket, replacing Parcel Resolvers: https://github.com/NASA-IMPACT/veda-ui/issues/896

We have Types (https://github.com/NASA-IMPACT/veda-ui/blob/main/parcel-resolver-veda/index.d.ts), utility functions (https://github.com/NASA-IMPACT/veda-ui/blob/main/parcel-resolver-veda/index.js#L211-L212) that are defined in VEDA Parcel resolver. Move these to somewhere Source code so it can be compiled with a library bundle.

## Acceptance Criteria

- [ ] Types and util functions don't have to be imported from 'veda' faux module. 

## Related Tickets
https://github.com/NASA-IMPACT/veda-ui/issues/896

**Comment by hanbyul-here** [comment]
Wooh~ I see a lot of works are already done in this PR (Thanks @oliverroick !) : https://github.com/NASA-IMPACT/veda-ui/pull/976/files#diff-4e01eeff5f76e67b260cae13bbf572e5c23eb56fa3ca04cc1a84c52f716288f6

**Comment by aboydnw** [comment]
@hanbyul-here @oliverroick should we pull this ticket into the current sprint and move to in progress? or is that PR just related, and not directly tied to this ticket?

**Comment by hanbyul-here** [comment]
Yeah, moving this ticket to the sprint makes sense. I am not sure if we can wrap up this sprint, but work is already being done.

**Comment by hanbyul-here** [comment]
Making a record that this ticket required merging the branch that @oliverroick has worked on to main since already quite an amount of work have been done on that branch. This branch had a library configuration needed to convert veda-ui from submodule to registry.
